openshift/release/release-notes.md

```diff
@@ -0,0 +1,376 @@
+
+  action required: Tasks and Pipelines with WhenExpressions that were created using Tekton Pipelines v0.16.x need to be reapplied to fix the case of their json annotations
+
+* :book: Fix the internal document link typo in docs/auth.md ([#3567](https://github.com/tektoncd/pipeline/pull/3567))
+
+* :book: Clarify input and output pipeline resource usage with PVC ([#3535](https://github.com/tektoncd/pipeline/pull/3535))
+
+  Added more detail to the documentation about input/output PipelineResources.
+
+* :book: Update docs when with version info ([#3528](https://github.com/tektoncd/pipeline/pull/3528))
+
+* :book: Fix markdown styling ([#3520](https://github.com/tektoncd/pipeline/pull/3520))
+
+* :book: Add a document to tell users how to configure Thread, QPS and Burst ([#3508](https://github.com/tektoncd/pipeline/pull/3508))
+
+* :book: Correct the disabled link ([#3507](https://github.com/tektoncd/pipeline/pull/3507))
+
+* :book: fixing broken links ([#3562](https://github.com/tektoncd/pipeline/pull/3562))
+
+* :book: Format workspaces description and correct unclear words ([#3547](https://github.com/tektoncd/pipeline/pull/3547))
+
+* :book: fix link to owners file in release readme ([#3538](https://github.com/tektoncd/pipeline/pull/3538))
+
+* :book: updating readme for 0.18.1 ([#3537](https://github.com/tektoncd/pipeline/pull/3537))
+
+* :book: Move the High Availability doc out of the developers subdirectory ([#3536](https://github.com/tektoncd/pipeline/pull/3536))
+
+* :book: updating readme with 0.18 ([#3511](https://github.com/tektoncd/pipeline/pull/3511))
+
+<hr>
+
+# Tekton Pipelines v0.18.1
+
+## Fixes
+
+* :bug: [cherry-pick] Fix recursion issue on Skip ([#3534](https://github.com/tektoncd/pipeline/pull/3534))
+
+  Fix and issue in the pipeline state resolution which lead to very long or failed start times and high controller CPU in case of pipelines with a large number of dependencies between tasks (40+).
+
+* :bug: [cherry-pick] Change the webhook name to pipeline-webhook ([#3533](https://github.com/tektoncd/pipeline/pull/3533))
+
+  Fix an issue that caused the webhook, under certain conditions, to fail to acquire a lease and not function correctly as a result.
+<hr>
+
+# Tekton Pipelines v0.18.0
+
+## Features
+
+* :sparkles: Add readiness and liveness probes in controller ([#3489](https://github.com/tektoncd/pipeline/pull/3489))
+
+* :sparkles: Init conformance test ([#3400](https://github.com/tektoncd/pipeline/pull/3400))
+
+  Initialize the conformance test
+
+* :sparkles: Add timestaps in logs ([#3392](https://github.com/tektoncd/pipeline/pull/3392))
+  
+  Add timestamp in the logs of the tekton pipelines controller and webhook
+
+* :sparkles: Replicate webhook pods for resiliency ([#3391](https://github.com/tektoncd/pipeline/pull/3391))
+  
+  Add HorizontalPodAutoscaler, PodDisruptionBudget and podAntiAffinity to webhook
```

Comment:
Same here, I think we do disable this in OSP right now, right ?
/cc @nikhil-thomas

Reply:
+1 👍

Reply:
Removed 
Thank you